services/web-app/pages/about-carbon/faq/index.mdx

```diff
@@ -1,15 +1,13 @@
 ---
 title: FAQs
-description:
-  What is the Carbon Design System? Who works on Carbon? How can I contribute? Here, we answer some
-  of our frequently asked questions.
+description: Below are frequently asked questions about the Carbon Design System.
```

Comment:
This description is what shows up in the pages metadata, not sure if "below..." text makes sense here? 

If you want it to show up on the page you can use the <PageDescription> component.

Reply:
Agreed this is probably not the description we want to go for!.
Also I think we need a `
<PageDescription>Below are frequently asked questions about the Carbon Design System.</PageDescription`

Because it's not showing up as per the figma design 
<img width="796" alt="image" src="https://user-images.githubusercontent.com/40550942/186946182-c70f3997-e5d8-4432-87ef-4cf39003458c.png">
<img width="760" alt="image" src="https://user-images.githubusercontent.com/40550942/186946216-6ed095d4-d3e6-4962-91a1-cac531d02c0c.png">

Reply:
✅ updated description and added page description component